payloads/library/execution/PolyWog-Shell/readme.md

```diff
@@ -0,0 +1,105 @@
+
+# PolyWog-Shell 
+### By: Hak5Peaks
+
+
+## PolyWog Shell Description. 
+
+Poly shell is a reverse shell generator designed for the Bash Bunny. This payload uses a python2 + bash script to generate a powershell payload with a unique signature. Every time the bash bunny is plugged into a computer, a unique powershell script will be injected into the target machine. 
+
+The python2 script uses a base powershell payload and randomizes/obfuscates parts of the powershell payload. Using methods such as variable and string Obfuscation and IP and Port Encoding
+
+## LED. 
+
+- Blue, payload has started and is running.
+- Red, Payload failed, most likely due to missing `shell.py`
+- Green, payload has completed. 
+
+
+## Configuations. 
+
+This script requires minimal configuration. simply launch a nc listener `nc -lvnp PORT` and copy the IP and Port from your listener into the IP and Port variable on the payload. Then place the python2 script `shell.py` inside of the `tools` directory of the udisk.
+ 
+
+### Shell + Payload configuation Example
+<div style="display: flex; justify-content: center; gap: 10px;">
+    <img src="https://github.com/user-attachments/assets/874226a5-6d3e-47d4-93bd-70fbf0ef5d2a" width="45%">
+    <img src="https://github.com/user-attachments/assets/7f656678-575e-43b3-8988-a60a70b01169" width="45%">
+</div>
+
+### Udisk configuration Example
+
+![image](https://github.com/user-attachments/assets/daf4b620-5d38-4ceb-86b0-95edafd0494d)
+
+
+## Requirements. 
+
+Ensure correct configuation as stated above. There is `no` other requirments or external packages that need to be install. 
```

Comment:
there _are_ no other

Reply:
make sure to comment somewhere if you have an SD card installed on the bunny, and if so the proper procedure for loading the payload onto the bunny in either case